recipes/xsel/build.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e -x
+
+autoreconf_args=(
+    --force
+    --verbose
+    --install
+    -I "${PREFIX}/share/aclocal"
+    -I "${BUILD_PREFIX}/share/aclocal"
+)
+autoreconf "${autoreconf_args[@]}"
+
+export CONFIG_FLAGS="--build=${BUILD}"
+
+export PKG_CONFIG_LIBDIR="${PREFIX}/lib/pkgconfig:${PREFIX}/share/pkgconfig"
+configure_args=(
+    ${CONFIG_FLAGS}
+    --disable-debug
+    --disable-dependency-tracking
+    --disable-selective-werror
+    --disable-silent-rules
+    --prefix="${PREFIX}"
+)
+
+if [[ "${CONDA_BUILD_CROSS_COMPILATION}" == "1" ]] ; then
+    configure_args+=(
+        --enable-malloc0returnsnull
+    )
+fi
+
+./configure "${configure_args[@]}"
+make -j${CPU_COUNT}
+make install
+
+rm -rf ${PREFIX}/share/man ${PREFIX}/share/doc/${PKG_NAME}
+
+# Remove any new Libtool files we may have installed. It is intended that
+# conda-build will eventually do this automatically.
+find ${PREFIX}/. -name '*.la' -delete
```

Comment:
Now that you deleted the initial comment about the libtool files, this has lost its relevant context & reference. Can you please re-add that link? The rest of the changes looks like a nice clean-up! 👍

Reply:
Completed. And I've put in your suggestions to the other six recipes as well.

Reply:
https://github.com/conda-forge/staged-recipes/pull/26163 https://github.com/conda-forge/staged-recipes/pull/26165 https://github.com/conda-forge/staged-recipes/pull/26166 https://github.com/conda-forge/staged-recipes/pull/26242 https://github.com/conda-forge/staged-recipes/pull/26254 https://github.com/conda-forge/staged-recipes/pull/26256 https://github.com/conda-forge/staged-recipes/pull/26333

I've synced your suggestions to the other recipes.